gradle/spotbugs.gradle

```diff
@@ -7,6 +7,26 @@ allprojects {
     omitVisitors = ["FindDeadLocalStores"]
     effort = "max"
     excludeFilter = file("$rootDir/gradle/spotbugs-exclude.xml")
+    toolVersion = '4.1.3'
+  }
+
+  spotbugsTest {
+    reports {
+      html {
+        enabled = true
+        destination = file("$buildDir/reports/spotbugs/main/spotbugs.html")
+        stylesheet = 'fancy-hist.xsl'
+      }
+    }
+  }
+  spotbugsMain {
+    reports {
+      html {
+        enabled = false
```

Comment:
First, why html report for tests is enabled and for other code disabled? Second, if this report is disabled, why do you configure its destination?

Reply:
Good catch, it shouldn't have been. Ideally, what I would like to do is, conditionally run HTML reports if not under a CI environment and XML otherwise for the CI server to process -  but I'm not sure if I can do this within the context of the script. I did notice some tests that picked up a system environment variable, maybe the same would work here?

Reply:
Ah I found it in the root gradle file! `def isCI = System.getenv("CI") != null`